lisa/tools/kill.py

```diff
@@ -21,9 +22,18 @@ def can_install(self) -> bool:
         return False
 
     def by_name(self, process_name: str, signum: int = SIGKILL) -> None:
-        running_processes = self.node.tools[Pgrep].get_processes(process_name)
-        for process in running_processes:
-            self.by_pid(process.id, signum)
+        pids = self.node.tools[Pidof].get_pids(process_name)
+        for pid in pids:
+            self.by_pid(pid, signum)
+        if not pids:
+            self.node.log.debug(
```

Comment:
Use the log object of self, instead of self.node. It affects the leading chars in this place. And the different log object maybe different on handlers.